.github/workflows/sync_branches.yml

```diff
@@ -8,6 +8,11 @@ on:
 
 permissions: {}
 
+env:
+  SYNC_COMMITTER_EMAIL: bot@mixxx.org
```

Comment:
This is a group email address. I have added your one as well, just in case.

Reply:
Yes, I believe this should be the email of @mixxxbot